scripts/pylib/twister/twisterlib.py

```diff
@@ -1075,6 +1075,12 @@ def _thread(handler, timeout, outdir, logfile, fifo_fn, pid_fn, results, harness
 
     def handle(self):
         self.results = {}
+        qemu_executable = self.instance.platform.filter_data.get("QEMU")
```

Comment:
in what situation quma binary is not exist or not executable?

Reply:
Previously, to get the SDK, I copied it directly from another machine. In which case  twister would hang forever without any error message, I spent a lot of time to find out the root cause ( I didn't try to run case with west), finanlly I found  qemu binary didn't work. So I think it's necessay to verify the qemu binary in twister

Reply:
in this case, it is a false operation, it is better to return error, not skip